test/unit/math/mix/fun/multiply_util.hpp

```diff
@@ -14,6 +18,46 @@ void instantiate_multiply() {
   Eigen::Matrix<std::complex<T>, -1, -1> cv_mat(2, 2);
   cv_mat << 1, 2, 3, 4;
 
+  auto d_scalar_d_mat = stan::math::eval(multiply(d_scalar, d_mat));
```

Comment:
[question]
Does this run autodiff and value tests?  If not, we should also have those tests.

Reply:
It doesn't because we currently do not support complex types in the ad testing framework but in a seperate PR I can add that. I think we just need to add a compile time flag to the ad testing scheme where if it's true then we also run the function with complex types